{"uuid": "c7501965-18aa-4d53-a9a4-b1734817533b", "children": ["c3986f0d-aa2b-4370-b2dc-903275a1ef2c"], "befores": [{"name": "setup_driver", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n", "trace": "  File \"C:\\Users\\prade\\PycharmProjects\\allgeo_web_app_pro\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\prade\\PycharmProjects\\allgeo_web_app_pro\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\prade\\PycharmProjects\\allgeo_web_app_pro\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\prade\\PycharmProjects\\allgeo_web_app_pro\\conftest.py\", line 57, in setup_driver\n    web_driver = webdriver.Chrome(executable_path=config.chromeDriver, options=chromeOptions)\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1712747751335, "stop": 1712747751335}], "start": 1712747751335, "stop": 1712747751361}